### PR TITLE
Render text/latex mime types using built in markdown serializer

### LINKED ISSUE #123144: Provide a builtin renderer for text/latex outputs
VS Code has a built in renderer for markdown in markdown cells
To have a consistent look and feel, I think VS Code should also provide a builtin renderer for  text/latex

Here's how users can generate latex (markdown) outputs in cells
* Install Jupyter & Python extensions
* Create an ipynb file
* Create a Python cell with the following content & run it (output mime is `text/latex`)
```
%%latex
Given : $\pi = 3.14$ , $\alpha = \frac{3\pi}{4}\, rad$
```
* Create a Python cell with the following content & run it (output mime is `text/latex`)
```
%%markdown
# Hello
```

The mime `text/latex` is not natively supported by VS Code event though `text/markdown` is.
Suggestion is to support `text/latex` just as we support `text/markdown` as VS Code supports late in mardow & text/latex is the same as `text/markdown` with latex.

FYI - Today the Jupyter extension ships with a renderer for text/latex

@rebornix @mattbierner /cc

### PR DESCRIPTION
**Pull request by DonJayamanne**
This PR fixes #123144

### CODE REVIEW

**Comment by DonJayamanne** [comment]
Screen Shot 2021-09-22 at 13 17 25

**Review by rchiodo — APPROVED** [review]

File: src/vs/workbench/contrib/notebook/common/notebookCommon.ts [hunk 1]
```diff
@@ -37,13 +37,15 @@ export const NOTEBOOK_DISPLAY_ORDER = [
 	'application/javascript',
 	'text/html',
 	'image/svg+xml',
+	Mimes.latex,
```
Comment (IanMatthewHuff) [inline src/vs/workbench/contrib/notebook/common/notebookCommon.ts:40]: This ordering actually looks off from the Jupyter ordering. I see this in classic notebooks and jupyterlab:
https://github.com/jupyter/notebook/blob/41f148395c2b056998768423257d9c8edb4244ec/notebook/static/notebook/js/outputarea.js#L272
https://github.com/jupyterlab/jupyterlab/blob/dbdefed9db9332381fe4104bdf53ec314451951f/packages/rendermime/src/factories.ts#L31

Comment (IanMatthewHuff) [reply src/vs/workbench/contrib/notebook/common/notebookCommon.ts:40]: I'm seeing markdown->latex->svg in that priority order if I'm reading those right. So SVG also looks misplaced.

**Review by IanMatthewHuff — CHANGES_REQUESTED** [review]
Code looks fine, but the ordering might be off? Just wanted to confirm the ordering before oking.

File: src/vs/workbench/contrib/notebook/common/notebookCommon.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/vs/workbench/contrib/notebook/common/notebookCommon.ts:40]: This isn't for ipynb files, there's a seprate ordering for ipynb notebooks.

**Review by IanMatthewHuff — APPROVED** [review]

**Pull request merged**